src/app/components/BookCard/index.tsx

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import { Text, View, Image } from 'react-native';
+import { Book } from '@interfaces/book';
 
 import styles from './styles';
 
 interface Props {
-  title: string;
-  author: string;
-  coverImageURI: string;
+  bookData: Book;
 }
 
-function BookCard({ title, author, coverImageURI }: Props) {
+function BookCard({ bookData: { title, author, imageUrl } }: Props) {
   return (
     <View style={styles.book}>
-      <Image source={{ uri: coverImageURI }} style={styles.bookCover} />
+      <Image
+        source={imageUrl ? { uri: imageUrl } : require('@assets/book_placeholder_cover.png')}
```

Comment:
Import this like the other variables.
like `import icCheck from '@assets/icCheck.png';`